1_fetch/src/calc_HIT.R

```diff
@@ -2,6 +2,7 @@
 
 
 calc_HITmetrics<-function(data,yearType,drainArea,floodThreshold){
+  drainArea <- drainArea$DA_NWIS
```

Comment:
Because drainage area is an argument to this function, where the function is called in targets.R we can change `drainArea=p1_drainage_area` to `drainArea=p1_drainage_area$DA_NWIS`. That way we don't have to modify this function.